.ci/pytorch/build.sh

```diff
@@ -191,7 +191,7 @@ fi
 
 # We only build FlashAttention files for CUDA 8.0+, and they require large amounts of
 # memory to build and will OOM
-if [[ "$BUILD_ENVIRONMENT" == *cuda* ]] && [[ "$TORCH_CUDA_ARCH_LIST" > "8.0" || "$TORCH_CUDA_ARCH_LIST" == "8.0" ]]; then
+if [[ "$BUILD_ENVIRONMENT" == *cuda* ]] && [[ "$TORCH_CUDA_ARCH_LIST" >= "8.0" ]]; then
```

Comment:
Somehow, `>=` is a wrong syntax when I test it out locally, thus the weird use of `>` and `==`.  Let's see

Reply:
Yeah, bash doesn't like this syntax and shows `syntax error in conditional expression`.  Let me fix this

Reply:
Let's see if `-ge` will do the trick